categorize.cfg

```diff
@@ -20,9 +20,10 @@ $idle > 60 ==> tag inactive,
 -- A rule that matches on a list of strings
 current window $program == ["Navigator","galeon"] ==> tag Web,
 
-current window $program == "sun-awt-X11-XFramePeer" &&
-current window $title == "I3P"
-  ==> tag Program:I3P,
+-- Use let-bindings to reduce duplication
+let isJava = current window $program ==
+  ["sun-awt-X11-XFramePeer", "sun-awt-X11-XDialogPeer", "sun-awt-X11-XWindowPeer"] in
+  $isJava && current window $title == "I3P" ==> tag Program:I3P,
```

Comment:
I think the syntax can still be improved a bit. The `let x = … in …` looks very much like a value binding, and people will try to write `let foo = current window $program` (and the line wrapping in this example makes this even more likely).

Is there a more specific keyword that would indicate that this constructs only binds, well, what does it bind? `Cond` according to the grammer. How about

    condition isJava = …
    in …

or 

    abbreviation isJava = …
    in …

Reply:
I like the `condition` syntax 👍 Updated the pr to use it